### PR TITLE
JNG-3176 Add workflows and update .xml files

### PR DESCRIPTION
**Pull request by FarkasValentin**

### CODE REVIEW

File: .flattened-pom.xml [hunk 1]
```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (robertcsakany) [inline .flattened-pom.xml]: This file have to be removed and added to .gitignore

File: features/.flattened-pom.xml [hunk 2]
```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (robertcsakany) [inline features/.flattened-pom.xml]: This file have to be removed and added to .gitignore

File: kar/.flattened-pom.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (robertcsakany) [inline kar/.flattened-pom.xml]: This file have to be removed and added to .gitignore

**Review by robertcsakany — CHANGES_REQUESTED** [review]

**Review by noherczeg — APPROVED** [review]

File: pom.xml [hunk 4]
```diff
@@ -98,169 +167,119 @@
...
+                    <id>judong-nexus-distribution</id>
+                    <!--suppress UnresolvedMavenProperty -->
+                    <url>https://nexus.judo.technology/repository/maven-judong-snapshots/</url>
                 </snapshotRepository>
                 <repository>
-                    <id>ossrh</id>
-                    <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+                    <id>judong-nexus-distribution</id>
+                    <!--suppress UnresolvedMavenProperty -->
+                    <url>https://nexus.judo.technology/repository/maven-judong-snapshots/</url>
```
Comment (noherczeg) [inline pom.xml:280]: This should be release no? @robertcsakany

**Review by noherczeg — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 5]
```diff
@@ -1,14 +1,18 @@
...
-    <version>1.0.1-SNAPSHOT</version>
+    <version>${revision}</version>
 
     <artifactId>apache-httpclient</artifactId>
     <name>Apache Http karaf feature project</name>
     <packaging>pom</packaging>
 
     <properties>
-        <karaf-version>4.2.5</karaf-version>
+        <revision>1.0.2-SNAPSHOT</revision>
```
Comment (noherczeg) [inline pom.xml:12]: Is this upgrade intentional? I'm not sure if this an established pattern.

**Review by noherczeg — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 4 above)
Comment (noherczeg) [reply pom.xml:280]: never mind

File: pom.xml (hunk 5 above)
Comment (noherczeg) [reply pom.xml:12]: never mind

**Review by noherczeg — APPROVED** [review]

**Pull request merged**